examples/TestReceiveVideo/Program.cs

```diff
@@ -0,0 +1,142 @@
+//-----------------------------------------------------------------------------
+// Filename: Program.cs
+//
+// Description: Demonstration of receiving and rendering video from a WebRTC
+// enabled browser using https://github.com/microsoft/MixedReality-WebRTC 
+// dotnet library.
+//
+// Author(s):
+// Aaron Clauson (aaron@sipsorcery.com)
+// 
+// History:
+// 07 Feb 2020	Aaron Clauson	Created, Dublin, Ireland.
+//
+// License: 
+// Licensed under the MIT License.
+//-----------------------------------------------------------------------------
+
+using System;
+using System.Drawing;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Microsoft.MixedReality.WebRTC;
+using WebSocketSharp;
+using WebSocketSharp.Net.WebSockets;
+using WebSocketSharp.Server;
+
+namespace TestNetCoreConsole
+{
+    public class SDPExchange : WebSocketBehavior
+    {
+        public event Action<WebSocketContext, string> MessageReceived;
+
+        public SDPExchange()
+        { }
+
+        protected override void OnMessage(MessageEventArgs e)
+        {
+            MessageReceived(this.Context, e.Data);
+        }
+    }
+
+    class Program
+    {
+        private const string WEBSOCKET_CERTIFICATE_PATH = "certs/localhost.pfx";

```

Comment:
This certificate file is not included. How does a user get/create it? Is it created by the code just below in `Main()`, or on the opposite does that code needs to load that certificate?